CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ SET(ProjectName cpp-blocks)
 
 PROJECT(${ProjectName})
 
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -W3")
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
```

Comment:
-Wall is too sensitive on Windows, so, ideally, this would use -Wall only for Linux.

Reply:
Changed to use -W3 only on MSVC. -Wall is default for "other" compilers. I believe this works with both LLVM and GCC.